src/libjpeg-turbo-2-install-DLL-to-bin-libjpeg-turbo.patch

```diff
@@ -0,0 +1,28 @@
+This file is part of MXE.
+See index.html for further information.
+
+From 703326874bf235adb392b9748ff0a56a08991690 Mon Sep 17 00:00:00 2001
+From: Boris Nagaev <bnagaev@gmail.com>
+Date: Sun, 6 Sep 2015 17:45:40 +0100
+Subject: [PATCH] install DLL to bin/libjpeg-turbo not to lib/bin
+
+---
+ ltmain.sh | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/ltmain.sh b/ltmain.sh
+index a968835..6acc338 100644
+--- a/ltmain.sh
++++ b/ltmain.sh
+@@ -5229,7 +5229,7 @@ fi\
+ 	  # place dlname in correct position for cygwin
+ 	  tdlname=$dlname
+ 	  case $host,$output,$installed,$module,$dlname in
+-	    *cygwin*,*lai,yes,no,*.dll | *mingw*,*lai,yes,no,*.dll) tdlname=../bin/$dlname ;;
++	    *cygwin*,*lai,yes,no,*.dll | *mingw*,*lai,yes,no,*.dll) tdlname=../../bin/libjpeg-turbo/$dlname ;;
```

Comment:
> DLL libjpeg-62.dll in bin/ can overwrite a file from package jpeg.

At least http://mxe.redjohn.tk/repos/apt/debian/pool/main/m/mxe-i686-w64-mingw32.shared-jpeg/mxe-i686-w64-mingw32.shared-jpeg_9a_amd64.deb doesn't have the file you are describing.

> Can you explain why DLLs are written to bin, please?

To be honest, I do not know. My guesses are:

1. dynamically linked exes can be directly executed if the DLLs are in the same directory as they are
2. historically you can't "link" to a DLL – only to its import library – which is the reason why import libs is installed to `lib/`